data/Spain/Congress/ep-popolo-v1.0.json

```diff
@@ -55616,8 +55092,10 @@
       "http://proyectocolibri.es",
       "http://www.congreso.es",
       "http://wikidata.org/",
-      "https://twitter.com/Congreso_Es/lists/diputados-2011-2016",
-      "https://twitter.com/Congreso_Es/lists/diputados-xi-legislatura/"
+      [
+        "https://twitter.com/Congreso_Es/lists/diputados-2011-2016",
+        "https://twitter.com/Congreso_Es/lists/diputados-xi-legislatura"
+      ]
```

Comment:
We’re currently blocked on https://github.com/everypolitician/everypolitician/issues/471 resolving this